### ISSUE TITLE
Azure Pipelines / runtime (Formatting windows x64) fails

### ISSUE THREAD

**Issue by buyaa-n** [dotnet/runtime#61331]
Azure Pipelines / runtime (Formatting windows x64) leg fails in https://github.com/dotnet/runtime/pull/61320:
```log
The process 'C:\hostedtoolcache\windows\Python\3.10.0\x64\python.exe' failed with exit code 1

Publishing build artifacts failed with an error: Not found PathtoPublish: D:\a\1\s\src\coreclr\format.patch
```
Error details:
```log
File "C:\hostedtoolcache\windows\Python\3.10.0\x64\lib\subprocess.py", line 1435, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified
##[error]The process 'C:\hostedtoolcache\windows\Python\3.10.0\x64\python.exe' failed with exit code 1
```
see more: https://dev.azure.com/dnceng/public/_build/results?buildId=1459308&view=logs&j=35546789-40c1-5b36-592c-13cb6b78e0dd&t=ee62566f-6723-5502-9bef-329d30faf067&l=115

**Comment by hoyosjs** [comment]
cc: @dotnet/jit-contrib

**Comment by AndyAyersMS** [comment]
Also failing for #61275

**Comment by BruceForstall** [comment]
This is due to my change https://github.com/dotnet/jitutils/pull/343 where I removed the batch file wrappers. I'll PR a fix in a moment.

**Comment by trylek** [comment]
Wow, great, thanks Bruce, I've just been tearing my hair out over my latest outerloop run where this failed too.

**Comment by BruceForstall** [comment]
> Wow, great, thanks Bruce, I've just been tearing my hair out over my latest outerloop run where this failed too.

Sorry for the trouble... I forgot to check the jit-format job with my jitutils change...

**Comment by AndyAyersMS** [comment]
Seems like we should finally fix jitutils to publish via nuget and gate updates on the runtime side just like other external dependencies...?

**Comment by BruceForstall** [comment]
> Seems like we should finally fix jitutils to publish via nuget and gate updates on the runtime side just like other external dependencies...?

Seems like we'd want to onboard to the whole arcade dependency flow world? There was some work to do this, now abandoned (https://github.com/dotnet/jitutils/pull/219). Or maybe there's a simpler way.

**Comment by am11** [comment]
> Or maybe there's a simpler way.

Simplest way is to use fixed commit ID (`fdf42ea`) rather than `main` (moving target) to avoid accidental breakages https://github.com/dotnet/runtime/blob/dca7f1043df2928b854de09fde078508cc9d4a3f/src/tests/Common/scripts/format.py#L138
